ledger/metrics/src/main/scala/com/daml/metrics/IndexDBMetrics.scala

```diff
@@ -18,10 +21,6 @@ trait TransactionStreamsDbMetrics {
   val prefix: MetricName
   val factory: MetricsFactory
 
-  @MetricDoc.GroupTag(
-    representative = "daml.index.db.flat_transactions_stream.<operation>",
-    groupableClass = classOf[DatabaseMetrics],
-  )
```

Comment:
How will this affect documentation? Previously all variables where grouped under the <operation> wildcard.

Reply:
I will have to double check, but the metrics should be unique by name, therefore only one instance will be actually output in the public docs, as expected. I will add the label doc as well.

Reply:
But will all the instances be present at docs (i.e. the names that the label can get)?

Reply:
![image](https://user-images.githubusercontent.com/107193850/222426730-0d6fed28-e4a8-4a2c-bc68-f7db1d05f236.png)

Reply:
@atriantafyllos-da No, the possible names will not be present but it open for debate as to if they should be present. With the labeled metrics it's really simple to explore the metrics by building ad hoc queries, therefore it's best to use the metrics themselves as a source to browse them.

Reply:
cc @mziolekda